src/lib/useDragAndZoom.tsx

```diff
@@ -37,20 +37,21 @@ export function fitBoundsInLimit(bounds: Bounds, limit: Bounds | undefined): Bou
     return bounds;
 }
 
-export function fitBoundsInMinVisibleX (newBounds: Bounds, oldBounds: Bounds, minVisibleX: number = 1): Bounds {
-    if (newBounds[1] - newBounds[0] > minVisibleX) {
-        return newBounds;
+export function fitBoundsInMinVisibleX (newBounds: Bounds, oldBounds: Bounds, minVisibleX: number | undefined): Bounds {
+    if (minVisibleX == null) {
+        minVisibleX = 1;
```

Comment:
Aah, sorry, now I understood why you had a default value here: because you might be calling this function with `undefined`, sorry :facepalm:

But I'm in doubt about `1` as a default value. Strictly speaking, anagraph could be not only used to display time-series graphs, but any X/Y graphs. And default value of `1` could be too much for general purpose graphs. But we certainly need some default value because we can't properly handle infinite zooming in with current implementation.

Can you please change default value to something like `1e-6`? I think it would be more reasonable for general-purpose graphs. And you can make it default value in function signature if you like. I was wrong about it because haven't noticed that you used default value for the case of passing `undefined`.

Sorry for bothering you with so many comments, this is a last one, I promise.

Reply:
I tried using 1e-6, but that value is too small, so I had to use 1e-3.

Reply:
That's because you are testing it on ZONT data which uses Unix-time with millisecond precision, which has 13 digits places before decimal point (`new Date().getTime().toString().length == 13`). But 64-bit IEEE 754 numbers with JS use has precision of about 16 decimal places. So, in case of time-series you only have about 3 digits after the decimal point. That's why you get strange behavior after zooming in deeper than 1e-3.

But 1e-3 is perfectly ok too :)